scene/gui/control.cpp

```diff
@@ -2270,7 +2270,8 @@ Control *Control::find_next_valid_focus() const {
 		ERR_FAIL_NULL_V_MSG(n, nullptr, "Next focus node path is invalid: '" + data.focus_next + "'.");
 		Control *c = Object::cast_to<Control>(n);
 		ERR_FAIL_NULL_V_MSG(c, nullptr, "Next focus node is not a control: '" + n->get_name() + "'.");
-		if (c->is_visible_in_tree() && c->get_focus_mode_with_override() != FOCUS_NONE) {
+		bool ac_enabled = c->get_tree() && c->get_tree()->is_accessibility_enabled();
+		if (c->is_visible_in_tree() && ((c->get_focus_mode_with_override() == FOCUS_ALL) || (c->get_focus_mode_with_override() == FOCUS_CLICK) || (ac_enabled && c->get_focus_mode_with_override() == FOCUS_ACCESSIBILITY))) {
```

Comment:
This repeats multiple times. It should be a method.

Reply:
Move it to `Control::_is_focusable()`.